migrate_nidirect_node/migrate_nidirect_node_nidirect_contact/migrate_nidirect_node_nidirect_contact.info.yml

```diff
@@ -10,3 +10,4 @@ dependencies:
   - drupal:migrate_plus
   - drupal:migrate_tools
   - migrate_nidirect_utils
+  - devel:devel
```

Comment:
What parts of devel do we rely on with this migration module?

Reply:
The devel dumper service to write dodgy telephone contact nids to the debug file.